data/json/recipes/other/containers.json

```
@@ -710,5 +710,18 @@
     "qualities": [ { "id": "HAMMER_FINE", "level": 1 }, { "id": "SCREW_FINE", "level": 1 }, { "id": "SAW_M_FINE", "level": 1 } ],
     "using": [ [ "welding_standard", 10 ], [ "steel_standard", 1 ] ],
     "components": [ [ [ "2lcanteen", 1 ] ] ]
+  },
+  {
+    "result": "bag_plastic",
+    "type": "recipe",
```

Since melting down a plastic chunk requires 10 plastic bags, this would be quite wasteful without a multiplier:
```suggestion
    "type": "recipe",
    "result_mult": 10,
```

An alternative would be using plastic scrap but we need to someday add like an uncraft to plastic chunks if we ever want to make those the primary material in a recipe, since they're not directly craftable currently.

---

Someone's gonna need to update Cata++ or mainline a recipe for plastic bags without the 3D printer I didn't even realize my recipe wasn't vanilla.